index.bs

```diff
@@ -9189,6 +9252,22 @@ for their contributions as our W3C Team Contacts.
     "date": "June 2019"
   },
 
+  "RFC9052": {
+    "authors": ["Jim Schaad"],
+    "title": "CBOR Object Signing and Encryption (COSE): Structures and Process",
+    "href": "https://datatracker.ietf.org/doc/rfc9052/",
+    "status": "IETF Internet Standard",
+    "date": "August 2022"
+  },
+
+  "RFC9053": {
+    "authors": ["Jim Schaad"],
+    "title": "CBOR Object Signing and Encryption (COSE): Initial Algorithms",
+    "href": "https://datatracker.ietf.org/doc/rfc9053/",
+    "status": "RFC Informational",
+    "date": "August 2022"
+  },
+
```

Comment:
These don't seem to be referenced anywhere in the added text; was this added by mistake?

Reply:
was required to make bikeshed run successfully

Reply:
How are you running Bikeshed? You may need to run `bikeshed update` to update your snapshot of the bibliography index.